docs/help/operator.md

```diff
@@ -801,6 +801,12 @@ spec:
     - globalMaxSize=512m
 ```
 
+## Providing additional brokerProperties configuration from a secret
+It is possible to replace the use of the activemqartemisaddresses CRD and much of the activemqartemissecurities CRD with configuration via broker properties. This can necessitate a large amount of configuration in the CR.brokerPropertis field.
+In order to provide a way to split or orgainse these properties by file or by secret, an extra mount can be used to provide a secret that will be treated as an additional source of broker properties configuration.
+Using an **extraMounts** secret with a suffix "-bp" will cause the operator to auto mount the secret and make the broker aware of it's location. In addition the CR.Status.Condition[BrokerPropertiesApplied] will reflect the content of this secret.
+Broker properties are applied in order, starting with the CR.brokerProperties and then with the "-bp" auto mounts in turn. Keys (or property files) from secrets are applied in alphabetical order.
```

Comment:
Does a test that checks `Keys (or property files) from secrets are applied in alphabetical order` already exist?

Reply:
yep, in the broker: https://github.com/apache/activemq-artemis/commit/7bc3b0280946811a65e6019f0e1ccba98097bf47#diff-c1a1b087900107abca96cc3dd98ef776ee58ebe4182024ac380ad4d06ba9f281